src/sagemaker/fw_utils.py

```diff
@@ -103,6 +103,17 @@
         "1.11.0",
     ],
 }
+
+PYTORCHDDP_SUPPORTED_FRAMEWORK_VERSIONS = [
+    "1.10",
```

Comment:
I don't see 1.10.2 here. Also are these versions essentially use the same DLC docker? which is 1.10.2? as we only have one DLC docker file for each minor release.

Reply:
Adding 1.10.2. 
Yeah, seems like there is only one DLC docker file for 1.10 (which is for 1.10.2): https://github.com/aws/deep-learning-containers/blob/master/available_images.md#prior-sagemaker-framework-container-versions